src/android/AccessibilityUtil.ts

```diff
@@ -11,9 +11,9 @@ import React = require('react');
 import RN = require('react-native');
 
 import Accessibility from '../native-common/Accessibility';
-import { AccessibilityUtil as CommonAccessibilityUtil } from '../native-common/AccessibilityUtil';
+import { NativeHelpers } from '../native-common/AccessibilityUtil';
```

Comment:
I think you have a potential circular dependency here. native-common/AccessibilityUtil -> android/AccessibilityUtil -> native-common/AccessibilityUtil